src/into_static.rs

```diff
@@ -6,7 +6,10 @@ use crate::{messages::*, HashMap, Prefix, Tags};
 ///
 /// This is used for going from a [`Message<'a>`](crate::messages::Message) to a `Message<'static>` or any of the sub types (see [`messages`](crate::messages))
 pub trait IntoStatic {
+    /// `Self` with a `'static` lifetime
```

Comment:
Technically, the output doesn't have to be 'Self'. it can by any type. 

`IntoStatic for str` could be `Output = String` or Output = `Cow<'static str>`

Its one of the reasons for not using ToOwned, so I can change the types if needs be.

Reply:
Aha, I'll fix that